sktime/forecasting/compose/_bagging.py

```diff
@@ -399,7 +399,13 @@ def get_test_params(cls):
 
         mbb = MovingBlockBootstrapTransformer(block_length=6)
         fcst = YfromX.create_test_instance()
-        params = [{"bootstrap_transformer": mbb, "forecaster": fcst}]
+        params_1 = {"bootstrap_transformer": mbb, "forecaster": fcst}
+
+        mbb_2 = MovingBlockBootstrapTransformer(block_length=12)
```

Comment:
thanks!

Could you change the transformer and the forecaster to be sth else?

Reply:
Sure @fkiraly !